dstep/translator/TypedefIndex.d

```diff
@@ -12,17 +12,19 @@ import clang.TranslationUnit;
 
 class TypedefIndex
 {
+    import std.typecons: Flag, No;
+
     private Cursor[Cursor] typedefs;
 
-    this(TranslationUnit translUnit)
+    this(TranslationUnit translUnit, Flag!"allTypedefs" allTypedefs = No.allTypedefs)
```

Comment:
Wouldn't it be better if this was a delegate instead, more of a filter?

Reply:
Maybe, but without an actual use case that would benefit from it, I'd tend to stick with the simplest thing that works.

Reply:
Let me rephrase this a bit. I think the `Flag` pattern is absolutely horrible and I would never accept that in DStep. A plain bool would not be very readable at the call site. A delegate would be more generic and void the problem with the bool.